src/app/(app)/project-sets/columns.tsx

```diff
@@ -0,0 +1,41 @@
+"use client"
+
+import {type ColumnDef} from "@tanstack/table-core"
+import {Checkbox} from "@/components/ui/checkbox"
+import {DataTableColumnHeader} from "@/components/ui/data-table-column-header"
+import {type ProjectSet} from "@/_temp_types/projects"
+
+export const columns: ColumnDef<ProjectSet>[] = [
+  {
+    id: "select",
```

Comment:
what are the bulk actions we currently envision for project sets?

Reply:
For now just deleting, but we haven't implemented that yet